File: Day_16/Q62.cpp
#include <bits/stdc++.h>
using namespace std;

//Input Size
int input_size(){

    int n;

    do{

        cout<<"Enter Size : ";
        cin>>n;

        if(n <= 0){
            cout<<"Invalid Size"<<endl;
        }

    }while(n <= 0);

    return n;

}

//Input Array
void input_array(vector<int> &arr){

    cout<<"Enter Elements"<<endl;

    for(int i = 0;i < arr.size();i++){
        cin>>arr[i];
    }

}

//Finding Maximum Frequency Element
int max_frequency(vector<int> &arr){

    int max_freq = 0;

    int ans = arr[0];

    for(int i = 0;i < arr.size();i++){

        int count = 0;

        for(int j = 0;j < arr.size();j++){

            if(arr[i] == arr[j]){
                count++;
            }

        }

        if(count > max_freq){
            max_freq = count;
            ans = arr[i];
        }

    }

    return ans;

}

int main(){

    int n = input_size();

    vector<int> arr(n);

    input_array(arr);

    cout<<"Maximum Frequency Element : "<<max_frequency(arr);

    return 0;
}

File: Day_16/Q62.java
import java.util.Scanner;

class Q62{

    //Input Size
    static int input_size(Scanner sc){

        int n;

        do{

            System.out.print("Enter Size : ");
            n = sc.nextInt();

            if(n <= 0){
                System.out.println("Invalid Size");
            }

        }while(n <= 0);

        return n;

    }

    //Input Array
    static void input_array(int arr[],Scanner sc){

        System.out.println("Enter Elements");

        for(int i = 0;i < arr.length;i++){
            arr[i] = sc.nextInt();
        }

    }

    //Finding Maximum Frequency Element
    static int max_frequency(int arr[]){

        int max_freq = 0;

        int ans = arr[0];

        for(int i = 0;i < arr.length;i++){

            int count = 0;

            for(int j = 0;j < arr.length;j++){

                if(arr[i] == arr[j]){
                    count++;
                }

            }

            if(count > max_freq){
                max_freq = count;
                ans = arr[i];
            }

        }

        return ans;

    }

    public static void main(String args[]){

        Scanner sc = new Scanner(System.in);

        int n = input_size(sc);

        int arr[] = new int[n];

        input_array(arr,sc);

        System.out.println("Maximum Frequency Element : " + max_frequency(arr));

        sc.close();
    }
}